lab_1/main.py

```diff
@@ -87,7 +143,30 @@ def get_adjacent_words(tokens: list, word: str, left_n: int, right_n: int) -> li
     right_n = 3
     --> [['man', 'is'], ['dog, 'cat']]
     """
-    pass
+    adjacent_words = []
+    words = []
+    if isinstance(tokens, list) and isinstance(word, str) and isinstance(left_n, int)\
```

Comment:
Usually, there are checks for arguments and body function - separated and on the same level.
Try to get rid of nesting.